server/src/test/java/io/spine/server/commandbus/AbstractCommandBusTestSuite.java

```diff
@@ -176,11 +190,42 @@ public void tearDown() throws Exception {
         eventBus.close();
     }
 
+    @Test
+    public void post_commands_in_bulk() {
+        final Command first = newCommand();
+        final Command second = newCommand();
+        final List<Command> commands = newArrayList(first, second);
+
+        commandBus.unregister(createProjectHandler);
+        commandBus.register(createProjectHandler);
+        final CommandBus spy = spy(commandBus);
+        spy.post(commands, StreamObservers.<Response>noOpObserver());
```

Comment:
Ideally we should check that `CommandStore.store(multipleCommands)` is called properly. But for that please see my general comment. 

And then we could probably review the decision of not making the `post` method `final` due to Mockito issues. From Mockito and proper API design I would choose the latter.